spec/requests/users_spec.rb

```diff
@@ -2,6 +2,29 @@
 
 RSpec.describe 'Users', type: :request do
   describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+    before(:example) { get '/users' }
+    it 'returns http success' do
+      expect(response).to have_http_status(:success)
+    end
+    it 'renders users index view' do
+      expect(response).to render_template(:index)
+    end
+    it 'shows content in the view' do
+      expect(response.body).to include('Here is a list of users')
+    end
+  end
```

Comment:
- [ ] This test is currently failing because you are expecting the `response.body` to include 'Here is a list of users' meanwhile the `response.body` includes 'Here is a list of user'. Please correct this 🙏

Reply:
Thanks for your help @codepantha 
I'll do that